torch/onnx/_internal/fx/op_validation.py

```diff
@@ -165,8 +182,28 @@ def validate_op_between_ort_torch(
                 diagnostic.level = diagnostics.levels.WARNING
 
 
+@_beartype.beartype
+def _convert_symint_to_int_in_shape(shape: torch.Size) -> torch.Size:
+    """Convert SymInt to int in shape"""
+    list_int_shape = []
+    for dim in shape:
+        if isinstance(dim, torch.SymInt):
+            if symbolic_shapes.has_hint(dim):
+                list_int_shape.append(symbolic_shapes.hint_int(dim))
+            else:
+                raise ValueError(
+                    f"Unexpected SymInt found in shape. SymInt: {dim}; "
```

Comment:
Is this unexpected, or is it a case we can expect but cannot handle? Just to make sure for clarity.

Reply:
nice catch, overlooked this one. Please reveal keyword "unbacked", will easily lead to many helpful resources such as this https://pytorch-dev-podcast.simplecast.com/episodes/unbacked-symints-oyqu0_P6

Reply:
Done